comet_core/api_helper.py

```diff
@@ -69,3 +70,29 @@ def decorated(*args, **kwargs):
         return f(*args, **kwargs)
 
     return decorated
+
+
+def valid_token(fingerprint, token):
+    """
+    Check if the token given in the request is valid by comparing
+    to the calculated API token.
+
+    Args:
+        fingerprint (str): the fingerprint to compute the API token with
+        token (str): the token to validate
+
+    Return:
+        True (bool): if the token is valid
```

Comment:
I think the syntax of this should be bool: this reading seems to me like it would always return true

Reply:
actually I think the syntax should be: token (bool): returns True is token is valid